README.md

```diff
@@ -0,0 +1,237 @@
+## OpenHardwareAPIs
+OpenHardwareAPIs - A set of Python APIs to control bare-metal switches fan,led,power,temperature sensors e.t.c.
+
+## Synopsis
+These APIs are written in python implemented by using ctypes to interface with C Language APIs provided by ONLP libraries.
+
+## Code Example 
+Following code is to redefine the “C” structures, in ONLP library, to python
+```
+class onlp_fan_info_t(ctypes.Structure):
+   _fields_ = [("hdr", onlp_oid_hdr),
+              ("status", ctypes.c_uint),
+              ("caps", ctypes.c_uint),
+              ("rpm", ctypes.c_int),
+              ("percentage", ctypes.c_int),
+              ("mode", ctypes.c_int),
+              ("model", ctypes.c_char * 64),
+              ("serial",ctypes.c_char * 64)]
+def get_fans():
+    id = 1
+    while(true):
+      fan = fan(id)
+      status = fan.status
+      if(status):
+         fanlist.append(fan)
+      else:
+         break
+      id = id + 1
+    return fanlist
+    
+class fan(object):
+    fanoid = 0x3000000
+    onlp_fan = onlp_fan_info_t()
+    def __init__(self, fanid):
+        self.fanoid = self.fanoid | fanid
+        self.libfan = ctypes.CDLL('/lib/x86_64-linux-gnu/libonlp.so')
```

Comment:
we can put dll loading into SDK init() so no need to load it each time create object.